src/PKSim.CLI/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
do we need **launchSettings.json** on GitHub?

Reply:
Nope